gum/backend-darwin/gumdarwinmodule.h

```diff
@@ -94,6 +101,8 @@ struct _GumDarwinModule
 
   GPtrArray * dependencies;
   GPtrArray * reexports;
+
+  GumDarwinModuleFlags flags;
```

Comment:
Should be moved right after `cache_file`, to follow the order that the properties are defined in.